src/main/java/bowling/domain/State.java

```diff
@@ -0,0 +1,27 @@
+package bowling.domain;
+
+public enum State {
+    STRIKE(),
+    SPARE(),
+    MISS(),
+    GUTTER(),
+    OTHER();
+
+    State() {}
```

Comment:
```suggestion
public enum State {
    STRIKE,
    SPARE,
    MISS,
    GUTTER,
    OTHER;
```

굳이 생성자를 미리 만들어주신 이유가 있으신가요?
다음 단계에서 State enum 에 멤버를 갖도록 할 예정인지 궁금하네요

Reply:
이넘을 많이 활용해본적이 없어서 항상 ()로 감싸줘야 되는줄 알고 있었네여;;